docs/sphinx/source/pvsystem.rst

```diff
@@ -0,0 +1,165 @@
+.. _pvsystemdoc:
+
+PVSystem
+========
+
+.. ipython:: python
+    :suppress:
+
+    import pandas as pd
+    from pvlib import pvsystem
+
+The :py:class:`~pvlib.pvsystem.PVSystem` class wraps many of the
+functions in the :py:mod:`~pvlib.pvsystem` module. This simplifies the
+API by eliminating the need for a user to specify arguments such as
+module and inverter properties when calling PVSystem methods.
+:py:class:`~pvlib.pvsystem.PVSystem` is not better or worse than the
+functions it wraps -- it is simply an alternative way of organizing
+your data.
+
+This guide aims to build understanding of the PVSystem class. It assumes
+basic familiarity with object-oriented code in Python, but most
+information should be understandable without a solid understanding of
+classes. Keep in mind that `functions` are independent of objects,
+while `methods` are attached to objects.
+
+See :py:class:`~pvlib.modelchain.ModelChain` for an application of
+PVSystem to time series modeling.
+
+
+.. _designphilosophy:
+
+Design philosophy
+-----------------
+
+The PVSystem class is designed to separate the data that represents a PV
```

Comment:
`is designed to include the data`

Reply:
I think I understand why you wanted to change this. I went with a different revision here so let me know if I've missed the intention.